weave/wandb_interface/wandb_lite_run.py

```diff
@@ -88,45 +121,46 @@ def i_api(self) -> InternalApi:
         return self._i_api
 
     @property
-    def run(self) -> Run:
+    def id(self) -> str:
+        return self._run_name
+
+    @property
+    def project(self) -> str:
+        return self._project_name
+
```

Comment:
Properties for `wandb.run` parity mainly for some testing stuff I was playing with.